src/cli/ProjectsCLIController.ts

```diff
@@ -4,32 +4,38 @@ import { BaseCLIController } from './BaseCLIController'
 import { getRepoConfig } from '../utils'
 
 export type Project = {
-  _id: string
-  name: string
+  _id?: string
+  name?: string
   key: string
 }
 
 export class ProjectsCLIController extends BaseCLIController {
+  public async getActiveProject() {
+    const stateProject = StateManager.getFolderState(this.folder.name, KEYS.PROJECT_ID)
+    if (stateProject) {
+      return stateProject
+    }
+    const repoConfig = await getRepoConfig(this.folder)
+    StateManager.setFolderState(this.folder.name, KEYS.PROJECT_ID, repoConfig.project)
+    return repoConfig.project
+  }
+
   public async getAllProjects() {
     const projects = StateManager.getFolderState(this.folder.name, KEYS.PROJECTS)
```

Comment:
If we're changing the structure of data that's stored here, we'll need to make sure that this handles parsing both the old and new data